google/cloud/logging_v2/logger.py

```diff
@@ -468,11 +478,12 @@ def commit(self, *, client=None, partial_success=True):
             client.logging_api.write_entries(
                 entries, partial_success=partial_success, **kwargs
             )
-        except InvalidArgument as e:
-            # InvalidArgument is often sent when a log is too large
-            # attempt to attach extra contex on which log caused error
-            self._append_context_to_error(e)
-            raise e
+        except Exception as e:
```

Comment:
same comment as above: hiding exceptions by default seems like it could have unintentional consequences, and could be a breaking change